unit_test.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"reflect"
+	"testing"
+	"encoding/json"
+
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func AreEqualJSON(s1, s2 string) bool {
+	var o1 interface{}
+	var o2 interface{}
+
+	var err error
+	err = json.Unmarshal([]byte(s1), &o1)
+	if err != nil {
+		panic(err)
+	}
+	err = json.Unmarshal([]byte(s2), &o2)
+	if err != nil {
+		panic(err)
+	}
+
+	return reflect.DeepEqual(o1, o2)
+}
+
+func Test_RetrieveSecret(t *testing.T) {
+	Convey("Given a valid JSON format stored secret", t, func() {
+		secret := `
+{
+ "a": 1,
+ "b": "xyz"
+}
+`
```

Comment:
Can you indent this to match var name? I'd assume that you can or at least trim it afterwards.